### DISCUSSION TITLE
docs: How do I ignore a field inside an array when using toObject?

### DISCUSSION THREAD

**Discussion by Alwinator** [mikro-orm/mikro-orm#3025] [Q&A] [answered]
I have tried using `.0`, but it did not work.
```ts
wrap(finalRecipe).toObject(['myarray.0.id'])
```

What is the right way to ignore fields inside a structure like this (I want to ignore `id: 4385`):
```ts

{
  id: 1,
  myarray: [
    {
      id: 4385,
      quantity: 2
    }
  ]
}
```

**Comment by B4nan** [comment, marked as answer]
How do you define your entity? You cant use partial loading for JSON properties, which sounds like what you want. The value is always read as a whole. 

If its just about serialization, you could either define custom `toJSON` method, or maybe you could use a property serializer that would return `undefined` (haven't tested this).

> wrap(finalRecipe).toObject(['myarray.0.id'])

The parameter is only for direct entity properties, no support for nesting, no support for dot notation.
